src/main/resources/static/js/gw.process.sidepanel.js

```diff
@@ -1,534 +1,720 @@
 /**
+
  * This file is dedicated to manage the prompt process dialog.
+
```

Comment:
Lot of linting changes. Hard to understand actual code changes.
Can you pls try to remove linting changes and push the actual code changes